tests/acceptance/SubscribeOnRegistrationPageCest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace MailPoet\Test\Acceptance;
+
+class SubscribeOnRegistrationPageCest {
+  function allowSubscribeOnRegistrationPage(\AcceptanceTester $I) {
+    $I->wantTo('Allow users to subscribe to lists on site registration page');
+    $I->login();
+    //Go to settings
+    $I->amOnMailPoetPage('Settings');
+    $I->seeInCurrentUrl('page=mailpoet-settings');
```

Comment:
I'm using these as assertions, and I don't notice a slowdown, but I don't think it's necessary. I pulled it.

Reply:
I understand it doesn't slow down a single test, but we have 70 of them, how much time does it take when we add up all of those?

Reply:
I can refactor to remove the others, I think I originally did it because it was in one of the acceptance tests I was using as a model, but the amOnMailPoetPage function doesn't need an assertion. 
I was saying "you're right" above. ;)